src/main/java/FeedbackSender.java

```diff
@@ -0,0 +1,54 @@
+import org.apache.commons.mail.DefaultAuthenticator;
+import org.apache.commons.mail.Email;
+import org.apache.commons.mail.EmailException;
+import org.apache.commons.mail.SimpleEmail;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class FeedbackSender extends CPServlet {
+
+    public void doPost(HttpServletRequest request,
+                       HttpServletResponse response)
+            throws ServletException, IOException
+    {
+
+        logger.info("---------User requested to send some feedback---------");
+
+        String feedbackMessage = (String) grabPropertyFromRequest("message", request);
+
+        logger.info("feedback message: " + feedbackMessage);
+        
+        boolean success = true;
+        try {
+            Email email = new SimpleEmail();
+            email.setHostName("smtp.googlemail.com");
+            email.setSmtpPort(465);
+            email.setAuthenticator(new DefaultAuthenticator("concordiacourseplanner@gmail.com", "tranzone"));
```

Comment:
shouldn't this be hidden from the public?

Reply:
In a perfect world ya. but I don't see a worthy method by which we can easily accomplish this.